src/pages/Mypage/PageBody/PageBodyApply/PageBodyApply.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+import ModalApply from "./ModalApply";
+import styled from "styled-components";
+import variables from "styles/variables";
+import { BiEditAlt } from "react-icons/bi";
+import { BiTrash } from "react-icons/bi";
+import { AiOutlineFileSearch } from "react-icons/ai";
+
+const PageBodyApply = () => {
+  const [isModal, setIsModal] = useState(false);
+
+  const applyView = (e) => {
+    setIsModal((cur) => !cur);
+  };
+
+  return (
+    <ul>
+      <ListItem>
+        <ListItemLeft>
+          <LeftTag>임시저장</LeftTag>
+          <LeftTitle>B마트 지점운영 Captain 모집(신입/경력)</LeftTitle>
+          <LeftTime>작성일 2022.10.06</LeftTime>
+        </ListItemLeft>
+        <ListItemRight>
+          <AiOutlineFileSearch
+            onClick={applyView}
+            size="24px"
+            className="icon"
+          />
+          <BiEditAlt size="24px" className="icon" />
+          <BiTrash size="24px" className="icon" />
+        </ListItemRight>
+        {isModal ? <ModalApply applyView={applyView} /> : null}
```

Comment:
```suggestion
        {isModal && <ModalApply applyView={applyView} />}
```
삼항연산자 보다는 && 연산자를 사용하는 것이 조금 더 좋겠네요!

Reply:
네 반영했습니다!